telcell/data/models.py

```diff
@@ -41,6 +41,25 @@ def __len__(self) -> int:
         return len(self.measurements)
 
 
+class MeasurementPair:
+    """
+    A pair of two measurements. The pair can be made with different criteria,
+    for example the time difference between the two measurements. It always
+    contains the information from the two measurements it was created from.
+    """
+    def __init__(self, track_a, track_b):
+        self.track_a = track_a
+        self.track_b = track_b
+
+        self.time_difference = abs(track_a.timestamp - track_b.timestamp)
+        # TODO track/sensor kolomnaam in extra omzetten naar owner/name
+        self.is_colocated = track_a.extra['track'] is not None and \
+            track_a.extra['track'] == track_b.extra['track']
+
+    def __str__(self):
+        return f"<{self.track_a}: ({self.track_b})>"
```

Comment:
Een komma lijkt me een logischer scheidingsteken dan een dubbele punt, dit suggereert asymmetrie die er volgens mij niet is.